index.html

```diff
@@ -45,5 +45,124 @@ <h2>LET'S CONNECT</h2>
+  </p>
+ <img src="./images/Tags.png" alt="" class="tag">
+      <div class="buttons">
+      <button class="press">See project</button>  
+      </div>
+</section>
+<section class="cardworks-four">
+  <img src="./images/Snapshoot Portfolio (4).png" alt="" class="snapshoot">
+  <div class="left-blockthree"></div>
+  <h6 class="headings">Multi-Post Stories</h6> 
+  <img src="./images/Frame 2.png" alt="" class="frame">
+  <p>
+  A daily selection of privately personalized reads; no accounts or sign-ups required. 
+  </p>
+  <img src="./images/Tags.png" alt="" class="tag">
+      <div class="buttons">
+       <button class="press">See project</button> 
+      </div>
+</section>
+</section>
+<section class="about-me">
+        <div>
+            <div class="about">About
+                Myself</div>
+                <p class="about-paragraph">Hello I’m a software developer! I can help you build a product , feature or  website Look through some of my work and experience! If you like what you see and  have a project you need coded,don’t hestiate to contact me.</p>
+                <h2 class="text">LET'S CONNECT</h2>
+                <ul class="tags">
+                    <li>
+                        <a href="#"><img src="./images/Vector (6).png" alt="github"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="./images/Vector (2).png" alt="linkedin"></a>
+                        <a href="#"><img src="./images/Vector (3).png" alt="linkedin"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img class="thumbs-up" src="./images/Vector (5).png" alt="thumbs-up"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="./images/Vector (1).png" alt="twitter"></a>
+                    </li> <li>
+                        <a href="#"><img src="./images/Vector (4).png" alt="Vector"></a>
+                    </li>
+                </ul>
+                <button type="submit" class="start">Get my resume</button>
+        </div>
+        <ul class="language">
+            <li>
+                <p class="languages">Languages</p>
+            </li>
+            
+        </ul>
+   <ul>
+        <li class="lists">
+          <a href="#"><img src="./images/Frame 9.png" alt="javascript"></a>
+        </li>
+         <li class="lists">
+          <a href="#"><img src="./images/Frame 8.png" alt="html"></a>
+         </li>
+         <li class="lists">
+          <a href="#"><img src="./images/Frame 11.png" alt="css"></a>
+         </li>
+         </ul> 
+         
+        <ul class="frameworks">
+            <li class="containe">
+               Frameworks 
+              <span class="arrow">&#62;</span>
+              </li>
+                
+                <li class="contain">
+            Skills
+              <span class="arrow">&#62;</span>  
+              </li>
+            
+        </ul>
```

Comment:
- [ ] I think  it would been better to have used a ``h3`` for the headings of the technologies or still using ``ul`` but give them a ``padding = 0;`` because padding have intrinsic padding that moves them to the right to address the problem addressed by the previous [reviewers](https://github.com/brendalutiku/Portfolio/pull/2/files#r1154867123)